analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -652,6 +652,9 @@ def main(args):
                   "when exporting to HTML.")
         sys.exit(1)
 
+    if export == 'gerrit' and gerrit.no_mandatory_env_var_is_set():
+        sys.exit(1)
+
```

Comment:
This does not print usage of this script. It would be better to use argument group. The above error maybe handled by argparser too.